business/src/main/java/in/koreatech/business/feature/store/OwnerStoreDialog.kt

```diff
@@ -0,0 +1,126 @@
+package `in`.koreatech.business.feature.store
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Card
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+import `in`.koreatech.business.R
+import `in`.koreatech.business.ui.theme.ColorDisabledButton
+import `in`.koreatech.business.ui.theme.ColorMinor
+import `in`.koreatech.business.ui.theme.ColorSecondary
+import `in`.koreatech.business.ui.theme.Gray6
+
+
+@Composable
+fun OwnerStoreDialog(
+    onDismissRequest: () -> Unit,
+    onConfirmation: () -> Unit,
+    dialogTitle: String,
+    dialogText: String,
+    positiveButtonText: String,
+    visibility: Boolean = false
```

Comment:
```suggestion
    dialogTitle: String,
    dialogText: String,
    positiveButtonText: String,
    visibility: Boolean = false,
    onConfirmation: () -> Unit,
    onDismissRequest: () -> Unit,
```
composable 함수 매개변수를 다음과 같은 규칙을 지향하여, 개선할 수 있습니다. (정답은 아니고, 이런 규칙을 지향한다..)
**"선택 매개변수 - 필수 매개변수 - 람다"**